ROADMAP.md

```diff
@@ -1,102 +1,126 @@
+* 👂🏽 <a href="https://github.com/aws/aws-cdk-rfcs/issues/228">CDK CLI Triggers</a> - CLI enhancements are also being considered for post command hooks. 
 
-A: CDK customers are making decisions and plans based on what we are developing. We strive to provide the 
-required information, when that is not sufficient, we take note of the feedback we receive and iterate on how 
-to bring improvements to our current processes and available information.
+### L2 Abstractions
+* 🔍 <a href="https://github.com/aws/aws-cdk-rfcs/issues/491">CloudFront Origin Access Control L2</a>
 
-**Q: How do you mark work in progress?**
+We are currently investigating other L2s to build out next. Feel free to create an RFC to request.
 
-A: For the [aws-cdk] repository, any issue that is currently worked on will have the `CDK Construct Squad` project 
-listed, with the current status.
+## Community Engagement
+We would love to hear from you on how the CDK operates today and how it should grow in the future. To report a bug or create a small feature request, please <a href="https://github.com/aws/aws-cdk/issues/new/choose">create an issue here</a>. If you are seeking to request a change in strategic direction or make a CDK core framework change, please <a href="https://github.com/aws/aws-cdk-rfcs/issues/new/choose">create a Request for Comments (RFC) ticket here</a>. If you are ever unsure about where your feature request should live, it is best to follow the first link within the aws-cdk repo.
 
-* **Needs Review** - We’re thinking about it, but cannot commit if, or when, we will work on items in this list.
-  This means we are still designing the feature and evaluating how it might work. This is the phase when we collect
-  customer use cases and feedback on how they want to see something implemented. There is no firm commitment to deliver
-  functionality listed in the Needs Review column, and there might be situations where we remove items from the Needs
-  Review column.
-* **Backlog** - We want to do this, but no one has picked it up yet. We have made an implied commitment to work on items
-  in this bucket, they have some level of design spec’ed out. Items might linger in this bucket as we work through the implementation details, or scope stuff out. Think several months out until a developer preview release, give or take.
-* **In Progress** - Someone on the CDK team is actively working on this. If all goes well, issues in this bucket should
-  be released in the coming weeks.
-* **In review** - It’s implementation is done and we are reviewing the code.
-* **Done** - It’s available now, fully supported by AWS.
+<img width="1174" alt="image" src="https://github.com/aws/aws-cdk/assets/142322013/ea006330-caa7-4c00-8eba-8e8fe379ef6b">
```

Comment:
This can also be done in markdown syntax, look at the raw markdown to see what i did: ![image](https://github.com/aws/aws-cdk/assets/142322013/ea006330-caa7-4c00-8eba-8e8fe379ef6b)